129.mediawiki

```diff
@@ -0,0 +1,85 @@
+<pre>
+BUIP129: Finish and Productize the BU voting system
+Proposer: Andrew Stone
+Submitted on: 2019-08-11
+Status: draft
+</pre>
+
+== Summary ==
+
+Currently the BU voting process simply requires that every user sign a vote. This BUIP allocates resources to evolve this process into a fully-fledged voting system that could be easily adopted by external organizations, integrated with or closely related to the Bitcoin Cash blockchain. This BUIP requests a person-year of work in funding to allow us to bring a developer on for a longer time frame than is normally possible via small projects. This should make this effort attractive to a larger set of possible candidates.
+
+== End Goal, User Perspective ==
+
+A website allowing an external entity to register and manage a vote. This website will be open source and easily cloned and customized by institutions that want a dedicated site.
+
+A mobile “Activity” (embeddable in any mobile app) that handles voting, connecting to any registered entity. Embed this activity into an app that integrates with the website described above.
+
+= Voting System Requirements =
```

Comment:
why not using `==` for this title section?

Reply:
No reason, was to quick when doing it.